library/java/test/io/envoyproxy/envoymobile/engine/EnvoyConfigurationTest.kt

```diff
@@ -21,9 +21,9 @@ class EnvoyConfigurationTest {
 
     val resolvedTemplate = envoyConfiguration.resolveTemplate(TEST_CONFIG)
     assertThat(resolvedTemplate).contains("domain: api.foo.com")
-    assertThat(resolvedTemplate).contains("connect_timeout: 123s")
-    assertThat(resolvedTemplate).contains("dns_refresh_rate: 234s")
-    assertThat(resolvedTemplate).contains("stats_flush_interval: 345s")
+    assertThat(resolvedTemplate).contains("connect_timeout: 123")
```

Comment:
Oops I meant to update all blocks: https://github.com/lyft/envoy-mobile/pull/451/files#diff-4f901a565c7644b56597246b16bf6ed2R10. The values _should_ have an `s`. We should add them to all the mock templates. [Here](https://github.com/lyft/envoy-mobile/blob/master/library/kotlin/test/io/envoyproxy/envoymobile/EnvoyClientBuilderTest.kt) and [here](https://github.com/lyft/envoy-mobile/blob/master/library/swift/test/EnvoyClientBuilderTests.swift) too.

Reply:
Ah, I did the inverse. Let me fix